bluesky/run_engine.py

```diff
@@ -1853,6 +1854,19 @@ async def _read(self, msg):
 
         return ret
 
+    async def _locate(self, msg):
+        """
+        Locate a Movable and return its location.
+
+        Expected message object is:
+
+            Msg('locate', obj)
```

Comment:
Actually, having the ability to locate multiple objects concurrently would be useful for load/save. Would this be `Msg('locate', *objs)`? Similarity with `set`, but we don't have `Status` objects, so would have to do the asyncio gather within this function

Reply:
I think extending this in the same way `set` does for multiple objects makes a lot of sense. I don't fully understand the implications of doing the asyncio gather here vs elsewhere.

Reply:
The `set()` method returns a `Status` object, so `Msg('set', obj)` only takes one device as an argument, and the combining of `Status` objects is done in the RE. `locate()` can be sync or async, and a `Status` object is not suitable as it is meant for long running tasks that don't have a return value. I think doing the grouping of locate objects in the message and doing the gather in this function is cleaner than pushing it up a level to the main RE loop like `set` does makes more sense here, but I'll defer to @tacaswell on this...

Reply:
`Msg("locate", obj1, obj2, ...)` - a single message with multiple objects seems appropriate, especially with the ability to use `gather()` and perform the requests in parallel.

Looking back at the `read` message (which is closer to `locate` than `set` is, in my opinion), args/kwargs to that message currently get passed to `obj.read(*args, **kwargs)`. However, that usage is *maybe* deprecated according to https://github.com/bluesky/bluesky/blob/e47d78f2285d82fb0bda5217b1f74e8a63fad370/bluesky/utils/__init__.py#L1754-L1763 This makes me think that a change of convention for `locate` is probably a good thing. Of course, @tacaswell may disagree...

Reply:
I agree.

Reply:
Are we happy with `Msg('locate', obj) -> obj_location` but `Msg('locate', obj1, obj2) -> [obj1_location, obj2_location]`?